tests/fail/unaligned_pointers/alignment.stderr

```diff
@@ -1,7 +1,7 @@
 error: Undefined Behavior: accessing memory with alignment ALIGN, but alignment ALIGN is required
   --> $DIR/alignment.rs:LL:CC
    |
-LL |         *(x_ptr as *mut u32) = 42;
+LL |         *(x_ptr as *mut u32) = 42; *(x_ptr        as *mut u32) = 42;
```

Comment:
I wonder how I managed to make the `.add(1)` disappear...

Reply:
it can't be one of the regexes, as those would either replace it or actually remove it, not replace with whitespace!?

Reply:
🤦